.changeset/early-mirrors-fry.md

```diff
@@ -0,0 +1,6 @@
+---
+"@yamada-ui/modal": minor
+"@yamada-ui/use-pan-event": patch
+---
+
+Close the Drawer by dragging
```

Comment:
Please separate the changeset (modal and use-pan-event).

Reply:
@taroj1205 
sorry, how can I separate changeset?

Reply:
You can remove one of the either like this.
```diff
---
"@yamada-ui/modal": minor
--"@yamada-ui/use-pan-event": patch
---
```

After you have removed the line, please run `pnpm changeset` and generate changeset for `@yamada-ui/use-pan-event`